src/Service/CloudFormation/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 0.3.0
+
+### Added
+
+- Enums; `Capability`, `ResourceStatus`, `StackDriftStatus`, `StackStatus`
+
+### Changed
+
+- Removed `requestBody()`, `requestHeaders()`, `requestQuery()` and `requestUri()` input classes. They are replaced with `request()`.
```

Comment:
Internal methods

Reply:
There were not internal at last release.

Reply:
:+1: